po/fr.po

```diff
@@ -16892,408 +16843,420 @@ msgstr "Changer le répertoire de travail"
+"fichier spécifié"
 
 #: /tmp/fish/implicit/share/completions/find.fish:19
 msgid "File last accessed specified number of days ago"
-msgstr ""
+msgstr "Le dernier accès au fichier remonte à il y a au plus n jours"
 
 #: /tmp/fish/implicit/share/completions/find.fish:20
 msgid "File status last changed specified number of minutes ago"
-msgstr ""
+msgstr "Le statut du fichier a été modifié il y a au plus n minutes"
 
 #: /tmp/fish/implicit/share/completions/find.fish:21
 msgid "File status last changed more recently than file was modified"
 msgstr ""
+"Le statut du fichier a été modifié plus récemment que le fichier spécifié"
 
 #: /tmp/fish/implicit/share/completions/find.fish:22
 msgid "File status last changed specified number of days ago"
-msgstr ""
+msgstr "Le statut du fichier a été modifié il y a au plus n jours"
 
 #: /tmp/fish/implicit/share/completions/find.fish:23
 msgid "File is empty and is either a regular file or a directory"
-msgstr ""
+msgstr "Le fichier est vide et un fichier standard ou un répertoire"
 
 #: /tmp/fish/implicit/share/completions/find.fish:24
 #: /tmp/fish/implicit/share/completions/test.fish:31
-#, fuzzy
 msgid "File is executable"
-msgstr "Exécutable"
+msgstr "Le fichier est exécutable ou le dossier est accessible"
 
 #: /tmp/fish/implicit/share/completions/find.fish:25
-#, fuzzy
 msgid "Always false"
-msgstr "Toujours"
+msgstr "Toujours faux"
 
 #: /tmp/fish/implicit/share/completions/find.fish:26
 msgid "File is on filesystem of specified type"
-msgstr ""
+msgstr "Le fichier est sur un système de fichier du type spécifié"
 
 #: /tmp/fish/implicit/share/completions/find.fish:27
 msgid "Numeric group id of file"
-msgstr ""
+msgstr "Le fichier a le groupe au GID spécifié en tant que groupe propriétaire"
 
 #: /tmp/fish/implicit/share/completions/find.fish:28
 msgid "Group name of file"
-msgstr ""
+msgstr "Le fichier a le groupe au nom spécifié en tant que groupe propriétaire"
 
 #: /tmp/fish/implicit/share/completions/find.fish:29
 msgid "File is symlink matching specified case insensitive pattern"
 msgstr ""
+"Le fichier est un lien symbolique vers un fichier dont le nom correspond au "
+"motif spécifié sans tenir compte de la casse"
 
 #: /tmp/fish/implicit/share/completions/find.fish:30
 msgid "File name matches case insensitive pattern"
-msgstr ""
+msgstr "Le nom du fichier correspond au motif sans tenir compte de la casse"
 
 #: /tmp/fish/implicit/share/completions/find.fish:31
-#, fuzzy
 msgid "File has specified inode number"
-msgstr "Utiliser la file spécifiée"
+msgstr "Le fichier a l’i-nœud dont le numéro est spécifié"
 
 #: /tmp/fish/implicit/share/completions/find.fish:32
 msgid "File path matches case insensitive pattern"
 msgstr ""
+"Le chemin absolu du fichier correspond à l’expression régulière sans tenir "
+"compte de la casse"
 
 #: /tmp/fish/implicit/share/completions/find.fish:33
 msgid "File name matches case insensetive regex"
```

Comment:
Typo in the English original.

Reply:
I don't understand what I should correct here; could you explain?

Reply:
Ha, that was not meant for you but for the fish maintainers – sorry, should have made that clearer.